SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +117,69 @@ etc...
-    
-For example:
+    For example:
 
-        "streaming_query": [
-          {
-            "global_component_streaming_query": {
-              "component_name": "improbable.unreal.GlobalStateManager"
-            }
-          },
-          {
-            "global_component_streaming_query": {
-              "component_name": "improbable.unreal.generated.yourprojectgamestate.YourProjectGameStateMultiClientRepData"
-            }
-          },
-          {
-            "global_component_streaming_query": {
-              "component_name": "improbable.unreal.generated.yourprojectgamemode.YourProjectGameModeMultiClientRepData"
+            "streaming_query": [
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.GlobalStateManager"
+                }
+            },
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.generated.yourprojectgamestate.YourProjectGameStateMultiClientRepData"
+                }
+            },
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.generated.yourprojectgamemode.YourProjectGameModeMultiClientRepData"
+                }
             }
-          }
-        ],
-        
+            ],
     
 ## Generating code for SpatialOS C++ workers and building your project
-1. Set your Unreal project to work with the SpatialOS Unreal GDK fork of the Unreal Engine, which you cloned and installed earlier. To do this:
-    1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-    1. Right-click your uproject file and select **Switch Unreal Engine version**.
-    1. Select the path to the Unreal Engine fork you cloned earlier.
-    1. In your terminal, from `Plugins\<UnrealGDKRoot>\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
-    1. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
-    1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
-
-## Running the Schema Generator and setting up the UE4 Editor
-1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Generate Schema** to run the `Schema Generator`.
-1. On the same toolbar, click **Snapshot**.
+Set your Unreal project to work with the SpatialOS GDK for Unreal fork of the Unreal Engine, which you cloned and installed earlier. To do this:
+1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
+2. Right-click your uproject file and select **Switch Unreal Engine version**.
+3. Select the path to the Unreal Engine fork you cloned earlier.
+4. In a terminal window, from `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`   
```

Comment:
Mention this step is optional, for cloud deployments only.

Reply:
This is for running locally managed workers, not cloud deployments.

Reply:
Mike is right, this is for the `Editor` config and so won't work in cloud. In our setup guide we include scripts for how to build cloud workers for client and server. Should we include those scripts here too?
Updated to 
```
This allows you to run locally managed workers. For example running a local spatial deployment with `one_worker` or `two_worker` launch configurations.
```